dtfs-central-api/src/cron-scheduler-jobs/delete-acbs-durable-function-logs/index.test.ts

```diff
@@ -0,0 +1,65 @@
+import { asString } from '@ukef/dtfs2-common';
+import { deleteAllAcbsDurableFunctionLogs } from '../../services/repositories/durable-functions-repo';
+import { deleteAcbsDurableFunctionLogsJob } from '.';
+import { WriteConcernError } from '../../errors';
+
+console.info = jest.fn();
+
+jest.mock('../../services/repositories/durable-functions-repo', () => ({
+  deleteAllAcbsDurableFunctionLogs: jest.fn(),
+}));
+jest.mock('@ukef/dtfs2-common', () => ({
+  asString: jest.fn(),
+}));
+
+describe('scheduler/jobs/delete-acbs-durable-function-logs', () => {
+  describe('the task', () => {
+    beforeEach(() => {
+      jest.mocked(asString).mockImplementation((value) => value as string);
+    });
+
+    afterEach(() => {
+      jest.resetAllMocks();
+    });
+
+    it('calls deleteAllAcbsDurableFunctionLogs from the durable functions repo', async () => {
+      // Arrange
+      jest.mocked(deleteAllAcbsDurableFunctionLogs).mockResolvedValue({ acknowledged: true, deletedCount: 1 });
+
+      // Act
+      await deleteAcbsDurableFunctionLogsJob.task(new Date());
```

Comment:
I'm a bit confused why this needs an argument passed in

Reply:
So, this is how it's going to actually be called by node-cron -- I agree its a bit odd to see in a unit test but this is the existing pattern.

This task eventurally gets passed into node.schedule, which then gets called with `date`

Hope this helps -- Took a bit of digging to find (and a chat with Josh to speed things along!)

Reply:
I've added a comment on the type for future reference